### PR TITLE
Addition of Vgp Workflows

### PR DESCRIPTION
**Pull request by Delphine-L**
Addition of 4 workflows for Genome assembly part of VGP pipeline v 2.0 : 
-   Creation of kmer count histograms with Meryl
-   Hifi Long Read phased assembly with Hifiasm
-   Scaffolding with Bionano Mapping data 
-   Scaffolding with HiC data

### CODE REVIEW

**Comment by simleo** [comment]
If you really want to follow semantic versioning, you should change the version from `0.1` to `0.1.0`, both in the `.ga` files (`"release"` field) and in the changelogs. Alternatively, you can delete the "and this project adheres to Semantic Versioning" statement from the Changelogs. I don't think it's required to be keepachangelog-compliant, and no workflow in IWC is currently doing that anyway.

File: workflows/VGP-assembly-v2/VGP-Bionano/Galaxy-Workflow-VGP_Bionano.ga [hunk 1]
```diff
@@ -14,206 +14,199 @@
     ],
     "format-version": "0.1",
     "license": "CC-BY-4.0",
-    "release": "0.1",
```
Comment (simleo) [inline workflows/VGP-assembly-v2/VGP-Bionano/Galaxy-Workflow-VGP_Bionano.ga]: This commit deletes the "release" field for three out of four workflows. It needs to be restored.

**Comment by bgruening** [comment]
>  .. ERROR: Non-optional input has no value specified in workflow test job [Is genome large (> 100 Mbp)?], job specifies inputs [['Bionano Data', 'Estimated genome size - Parameter File', 'Hifiasm Purged Assembly', 'species_full_name', 'species_id', 'version']]
.. ERROR: Non-optional input has no value specified in workflow test job [Type of organism], job specifies inputs [['Bionano Data', 'Estimated genome size - Parameter File', 'Hifiasm Purged Assembly', 'species_full_name', 'species_id', 'version']]

I see this error in the logs.

File: workflows/VGP-assembly-v2/VGP-Bionano/Galaxy-Workflow-VGP_Bionano.ga [hunk 2]
```diff
@@ -0,0 +1,625 @@
...
+            "type": "tool",
+            "uuid": "6a8fcbb8-5c90-48a2-a99c-b78f536109f3",
+            "workflow_outputs": [
+                {
+                    "label": "Quast on Bionano scaffolding: Tabular",
+                    "output_name": "quast_tabular",
+                    "uuid": "41344b83-8cd1-4f46-8512-58f5f724fa3f"
+                },
+                {
+                    "label": "Quast on Bionano scaffolding: PDF",
```
Comment (gallardoalba) [inline workflows/VGP-assembly-v2/VGP-Bionano/Galaxy-Workflow-VGP_Bionano.ga:605]:
```suggestion
                    "label": "Quast on Bionano scaffolding: PDF report",
```

**Comment by gallardoalba** [comment]
Meryl workflows have been included already https://github.com/galaxyproject/iwc/tree/main/workflows/VGP-assembly-v2 So I think can be removed form this PR.

File: workflows/VGP-assembly-v2/README.md [hunk 3]
```diff
@@ -1,28 +1,288 @@
...
+### Import workflows to Galaxy
 
-1. Phased assembly export
-2. Purged assembly export
-3. Bionano scaffolding export
-4. Hi-C Scafolding export
+> Note : You need to be a registered user in a Galaxy instance to use workflows.
+
+
+To import the VGP workflows into Galaxy, download the `.ga` files on your computer from this github repository.
```
Comment (lldelisle) [inline workflows/VGP-assembly-v2/README.md:16]: For me this step is not easy for the user. What about suggesting to go to [dockstore](https://dockstore.org/organizations/iwc/collections/vgp), click on the workflow they want and use the icon galaxy to import the workflow in their prefered galaxy instance?

**Pull request closed without merging**